modules/apps/document-library/document-library-external-video/src/main/java/com/liferay/document/library/external/video/internal/constants/ExternalVideoWebKeys.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.document.library.external.video.internal.constants;
+
+/**
+ * @author Alejandro Tardín
+ */
+public class ExternalVideoWebKeys {
+
+	public static final String EMBEDDED_HTML = "EMBED_HTML";
```

Comment:
Shouldn't this be named "EMBED_HTML"? Or alternatively, rename the constant value to "EMBEDDED_HTML".